src/KubeOps/KubeOps.csproj

```diff
@@ -28,6 +26,7 @@
         <PackageReference Include="KubernetesClient" Version="2.0.29" />
         <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.0.0" />
         <PackageReference Include="McMaster.Extensions.Hosting.CommandLine" Version="3.0.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="3.1.7" />
```

Comment:
I think this one was missed when splitting it out to the Testing project

Reply:
whoops. you're right.

Reply:
well at least it's not that grave. We can remove it in another branch

Reply:
Sorry I missed that prior to you merging :-) Not a biggie though, fortunately